src/test/kotlin/org/malachite/estella/BaseIntegration.kt

```diff
@@ -269,7 +272,7 @@ class BaseIntegration {
             (this["dateTime"] as String?)?.toTimestamp(),
             this["minutesLength"] as Int,
             (this["application"] as Map<String, Any>).toApplicationDTO(),
-            this["hosts"] as List<String>?,
+        (this["hosts"] as List<String>?),
```

Comment:
nie zmieniłeś hostów na set? wiem, że to pewnie przy takiej deserializacji nie ma znaczenia, ale still.